### PR TITLE
Missing WS connection fallback for liveview

### PR DESCRIPTION
**Pull request by blazoncek**
- fix for #3250

Remove (conditional WLED_ENABLE_LEGACY) legacy URI

Replace /sliders with /?sliders (easier reverse proxy handling)

### CODE REVIEW

**Review by Aircoookie — APPROVED** [review]
The fallback did not work as `/json/live` was not compiled in by default. Also merged the two liveview pages, which saves 0.5kB and thus fully offsets the added binary size of enabling `/json/live`. Now ready to merge IMO!

**Comment by blazoncek** [comment]
There's one more thing I noticed while testing this while I was at work: WS (from browser) keeps trying to connect. Perhaps we could add a counter to stop retries after a few attempts and only repeat 1 try after each HTTP request.
What do you think?

File: wled00/data/index.js [hunk 1]
```diff
@@ -1697,7 +1698,7 @@ function toggleLiveview()
 	}
 
 	gId(lvID).style.display = (isLv) ? "block":"none";
-	gId(lvID).src = (isLv) ? getURL("/" + lvID + ((wsOn) ? "?ws":"")):"about:blank";
+	gId(lvID).src = (isLv) ? getURL("/" + lvID):"about:blank";
```
Comment (Aircoookie) [inline wled00/data/index.js]: Why did you remove this? liveview uses the `?ws` parameter to determine if it can use WS, without it it will always fallback to json.

**Comment by Aircoookie** [comment]
WS retry looks good. Also added an attempt reset when the WS connection is opened, since preset loading only happens on page load or after presets were modified.

File: wled00/data/index.js (hunk 1 above)
Comment (blazoncek) [reply wled00/data/index.js]: I am sorry. I missed the "?" in the string.
